lib/pack/utils.go

```diff
@@ -722,6 +722,13 @@ func IsPlanetConfigPackage(loc loc.Locator, labels map[string]string) bool {
 		loc.Repository != defaults.SystemAccountOrg
 }
 
+// IsMetadataPackage determines if the specified package is a metadata package.
+// A metadata package describes a remote package and deserves
+// special handling in certain cases.
```

Comment:
Nit: TBH "special handling in certain cases" doesn't tell the reader much :smile: I'd either try to be more specific or remove this phrase altogether.